### PR TITLE
Note to Clarity Connection Differences

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2157

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-in-teams.md [hunk 1]
```diff
@@ -315,7 +315,8 @@ These settings control which meeting participants wait in the lobby before they
 - [Allow dial-in users to bypass the lobby](#allow-dial-in-users-to-bypass-the-lobby-coming-soon)
 - [Allow organizers to override lobby settings](#allow-organizers-to-override-lobby-settings-coming-soon)
 
-### Automatically admit people
+> [!NOTE]
+> Joining a meeting will vary depending on the settings for each Teams group and the method of connecting. If your group has audio conferencing and you are connecting through audio conferencing, then refer to [Audio Conferencing in Office 365](https://docs.microsoft.com/microsoftteams/audio-conferencing-in-office-365). If your Teams group does not have audio conferencing, then refer to [Join a meeting in Teams](https://support.office.com/article/join-a-meeting-in-teams-1613bb53-f3fa-431e-85a9-d6a91e3468c9).
```
Comment (nenonix) [inline Teams/meeting-policies-in-teams.md]:
```suggestion
### Automatically admit people
```

File: Teams/meeting-policies-in-teams.md [hunk 2]
```diff
@@ -315,7 +315,8 @@ These settings control which meeting participants wait in the lobby before they
...
```
Comment (nenonix) [inline Teams/meeting-policies-in-teams.md]:
```suggestion
>Options to join a meeting will vary, depending on the settings for each Teams group, and the connection method. If your group has audio conferencing, and uses it to connect, see [Audio Conferencing in Office 365](https://docs.microsoft.com/microsoftteams/audio-conferencing-in-office-365). If your Teams group does not have audio conferencing, refer to [Join a meeting in Teams](https://support.office.com/article/join-a-meeting-in-teams-1613bb53-f3fa-431e-85a9-d6a91e3468c9).
```

**Comment by TokyoScarab** [comment]
Yea, sorry about that, that line got removed by mistake.  Should be all good now

**Pull request merged**